scripts/india_wris/util/groundWater.json

```diff
@@ -0,0 +1,122 @@
+[
+    {
+        "Pollutant": [
+            {
+                "name": "Arsenic (As)",
+                "statvar": "Arsenic",
+                "unit": "dcid:MicrogramsPerLitre"
```

Comment:
Nit: seems inconsistent to add the dcid prefix for unit, but not the statvar. Choose one -- it's probably easiest to omit it here, and add it programatically if the unit is added.

Reply:
I implemented this.